### PR TITLE
Preserve ids

### PR DESCRIPTION
**Pull request by hyperknot**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,11 +8,15 @@ function flatten(gj) {
...
             if (!gj.geometry) return gj;
             return flatten(gj.geometry).map(function(geom) {
-                return {
+                var data = {
                     type: 'Feature',
                     properties: JSON.parse(JSON.stringify(gj.properties)),
                     geometry: geom
-                };
+                }
+                if (gj.id || gj.id === 0) {
```
Comment (tmcw) [inline index.js]: `gj.id  !== undefined` would let you do this check in one step.

**Comment by hyperknot** [comment]
True, that's nicer. Do we care about null?

**Comment by tmcw** [comment]
Nope.

**Comment by hyperknot** [comment]
Done

**Pull request merged**